implementations/prometheus_benchmark/compression_benchmark_test.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// Set up metrics with random seed for reproducibility
+var r = rand.New(rand.NewSource(12345))
```

Comment:
rand.Seed is a noop without a specific go build directive as of go 1.24, this is the recommended way to get a seeded rand.